chia/consensus/block_body_validation.py

```diff
@@ -154,24 +154,42 @@ async def validate_block_body(
     if height > constants.INITIAL_FREEZE_PERIOD and constants.NETWORK_TYPE == NetworkType.MAINNET:
         return Err.INITIAL_TRANSACTION_FREEZE, None
     else:
-        # 6. The generator root must be the tree-hash of the generator (or zeroes if no generator)
+        # 6a. The generator root must be the hash of the serialized bytes of
```

Comment:
Is the generator_root and generator_ref_list root supposed to be the hash of the item in the current block, or a previous block?

Reply:
Current one